superset-frontend/src/views/CRUD/alert/types.ts

```diff
@@ -52,7 +52,7 @@ export type AlertObject = {
   database?: MetaObject;
   description?: string;
   grace_period?: number;
-  id?: number;
+  id: number;
```

Comment:
Does this not throw a ts error when creating alerts?

Reply:
I updated the state type to be `Partial<AlertObject>` so that we can store incomplete AlertObjects in state.